skore/src/skore/item/cross_validation_item.py

```diff
@@ -106,14 +114,15 @@ def factory(cls, reporter: CrossValidationReporter):
         if not isinstance(reporter, CrossValidationReporter):
             raise ItemTypeError(
                 f"Type '{reporter.__class__}' is not supported, "
-                "only 'CrossValidationReporter' is."
+                f"only '{CrossValidationReporter.__name__}' is."
```

Comment:
Why this change? `'CrossValidationReporter' == CrossValidationReporter.__name__`.

Reply:
If the class ever changes names, the hard-coded string could be forgotten